04-rstudio-tour.Rmd

```diff
@@ -1,38 +1,277 @@
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g22370a5f292_430_5")
+```
+
+</details>
+
+
+The pane on the left (labeled "Pane 1" in the image) is where we write code.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The pane on the left is where we write and test code."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_81")
+```
+
+The pane on the top right (labeled "Pane 2" in the image) is where we can see what objects we have created and are actively in memory (meaning they can be used at that time) in what is called the "Environment".
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The pane on the upper right is where we see our objects."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_95")
+```
+
+
+The pane on the bottom right (labeled "Pane 3" in the image) is where we can find files on computer (the "Files" tab), see plots (the "Plots tab), and get coding help (the "Help tab).
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Whe pane on the lower right is where we see our files, plots, and where we get help."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_109")
+```
+
+
+While there are other tabs, don't worry about those for now. We will go deeper into RStudio as we continue. 
+
+
+Let's try an example to get started.
+
+As an example, we could type in the code `head(iris)` and press enter to see the code execute and preview.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "Running some code in RStudio"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g20ed7630a13_1_142")
+```
+
+Now let's try another example where instead of just printing some data to the screen we assign a data object that will show up in the environment using the `<-` notation. This is useful in a situation if we want to modify the iris data somehow but want to keep the original version.
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "If you create an object in RStudio, it will show up in the environment pane."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g1fa1583c827_0_4")
+```
+
+### The Hidden Pane
+
+There is a hidden fourth pane. This is only accessible when we start to make a script or a report with our code.
```

Comment:
```suggestion
There is a hidden fourth pane. This is only accessible when we start to make a script or a report with our code. 

In order to make our analysis truly reproducible we will need **every single step** written down in script or notebook. This is why using the Console is great for testing things, but not so great for actually performing your analysis. 

We are going to instroduce you to a file that really helps make analyses reproducible: an R Markdown file. R Markdown files allow you to have the code for your analysis, the output from the analysis (so plots and stats, etc) and your written thoughts and rationale for your analysis all in one place! This makes it a snap to share your analysis with others in a reproducible way!
```

Reply:
We should add a slide here of the parrot saying how great notebooks are because they have all these components.

Reply:
I have one of these in the next chapter do you think we should use it in both places? I give more context in the next chapter.

Reply:
I ended up splitting the text you suggested up and moving it around a tad but I added it.